packages/chart/src/ChartModelFactory.ts

```diff
@@ -1,4 +1,4 @@
-import dh, { Figure, Table } from '@deephaven/jsapi-shim';
+import { dhType, Figure, Table } from '@deephaven/jsapi-shim';
```

Comment:
Import types from `@deephaven/jsapi-types` instead, e.g.:
```suggestion
import type { dh as dhType, Figure, Table } from '@deephaven/jsapi-types';
```

We should be able to remove dependency on `@deephaven/jsapi-shim` entirely.